src/app/features/genes/components/gene-comparison-tool/components/gene-comparison-tool-filter-list-item/gene-comparison-tool-filter-list-item.component.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  Component,
+  Input,
+  Output,
+  EventEmitter,
+  ViewEncapsulation,
+} from '@angular/core';
+
+@Component({
+  selector: 'gene-comparison-tool-filter-list-item',
+  templateUrl: './gene-comparison-tool-filter-list-item.component.html',
+  styleUrls: ['./gene-comparison-tool-filter-list-item.component.scss'],
+  encapsulation: ViewEncapsulation.None,
+})
+export class GeneComparisonToolFilterListItemComponent {
+  @Input() item: any;
```

Comment:
The item that will be sent to the parent when the chiclet remove button is clicked, which is used when clearing the GCT filters -- if there's a better way to handle this, happy to adjust!